src/main/java/core/basesyntax/db/report.csv

```diff
@@ -0,0 +1 @@
+{banana=152, apple=90}
```

Comment:
File foormat doesn't comply with requirement
in task description:
    fruit,quantity
    banana,152
    apple,90